### PR TITLE
cowpatty 4.6

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

File: Library/Formula/cowpatty.rb [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 
   def install
-    ENV.j1
-    inreplace "Makefile", "/usr/local/", "#{prefix}/"
-    system "make install"
+    system "make", "BINDIR=#{bin}", "install"
+  end
+
+  test do
+    system bin/"cowpatty", "-h"
```
Comment (mikemcquaid) [inline Library/Formula/cowpatty.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by dunn** [comment]
OK, got a real test in there.

**Comment by asparagui** [comment]
bi cowpatty
==> Downloading http://www.willhackforsushi.com/code/cowpatty/4.6/cowpatty-4.6.t
######################################################################## 100.0%
==> make BINDIR=/usr/local/Cellar/cowpatty/4.6/bin install
clang: error: no such file or directory: 'utils.o'
clang: error: no such file or directory: 'sha1.o'
make: *** [cowpatty] Error 1
make: *** Waiting for unfinished jobs....
make: *** [genpmk] Error 1

**Comment by dunn** [comment]
add `ENV.j1` back in?

**Comment by asparagui** [comment]
@dunn I have j1 here, doesn't seem to change things. :-|

**Comment by asparagui** [comment]
I think this might be a libpcap (or random libressl) linking issue.

**Comment by jacknagel** [comment]
There is a test.pcap file available as `test_fixtures("test.pcap")`, can we use that instead of downloading a file?

**Comment by dunn** [comment]
@jacknagel awesome, I didn't know that.  Anything else I should change at the same time that might help with @asparagui's issue?

**Comment by dunn** [comment]
Thought this was #38865. For that PR I can use the test fixture, but I think cowpatty needs to know the SSID associated with the packet capture data. Does the test fixture have SSID metadata?

**Comment by asparagui** [comment]
It's also possible my system is acting up. I think the test is fine.

**Comment by mikemcquaid** [comment]
Anyone object if I just pull this?

File: Library/Formula/cowpatty.rb [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
+  end
 
   def install
-    ENV.j1
-    inreplace "Makefile", "/usr/local/", "#{prefix}/"
-    system "make install"
+    system "make", "BINDIR=#{bin}", "install"
+  end
+
+  test do
```
Comment (asparagui) [inline Library/Formula/cowpatty.rb]: :tada:

**Comment by asparagui** [comment]
Pulling, thanks!

**Pull request closed without merging**